src/register/register.ts

```diff
@@ -214,13 +223,13 @@ export class Register {
    *
    * `REMARKS:` This Procedure assume that you pass an valid register.
    */
-  private static putNormalRegister(
+  private static async putNormalRegister(
     content: RegisterContent,
     register: string,
     vimState: VimState
-  ): void {
+  ): Promise<void> {
     if (Register.isClipboardRegister(register)) {
-      Clipboard.Copy(content.toString());
+      await Clipboard.Copy(content.toString());
```

Comment:
I don't think we need to await this?

Reply:
Why not? How do we make sure what is in the clipboard is what we intend to be in the clipboard then?